lefthook.yml

```diff
@@ -0,0 +1,14 @@
+pre-push:
+  commands:
+    coverage:
+      glob: '*.{js,ts,jsx,tsx}'
```

Comment:
Let's pick the entries we currently have: js, ts, md

Reply:
d1